docs/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ~~~~~~~~~
 
+0.29.0 (June 2023)
+------------------
```

Comment:
```suggestion
---------------------

Reply:
Oh man, I'll never learn rst 🙈

Reply:
That is not a mistake you have done, that is the needed extension because of the version change I suggested